libs/variant/include/variant/variant.hpp

```diff
@@ -158,17 +158,17 @@ class Variant
   /// @name Operators
   /// @{
   template <typename T>
-  meta::IfIsBoolean<T, Variant &> operator=(T const &value);
+  meta::IfIsBoolean<T, Variant &> operator=(T const &value);  // NOLINT
```

Comment:
What is being nolinted here

Reply:
Clang-tidy can't get past the SFINAE - complains about a non-standard assignment operator which does not return a reference